client/shared/record_store/base_records_interface.coffee

```diff
@@ -4,22 +4,26 @@ utils         = require './utils'
 module.exports =
   class BaseRecordsInterface
     model: 'undefinedModel'
+    collectionName: null
```

Comment:
I don't think we're using this option anymore? Something would seem quite off to me if we were making a db table that wasn't the plural of the model.